network/wsNetwork_test.go

```diff
@@ -53,6 +53,11 @@ import (
 
 const sendBufferLength = 1000
 
+func init() {
+	// this allows test code to use out-of-protocol message tags and have them go through
+	allowCustomTags = true
```

Comment:
Are we confident that go will never load this test package in production? 
Why not keep this false, and whichever test that needs this feature, can set it to true then false, and not run in parallel.